{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'MAF': None,\n",
      "  'ambiguity': 'S',\n",
      "  'clinical_significance': ['pathogenic'],\n",
      "  'evidence': ['Cited', 'Phenotype_or_Disease'],\n",
      "  'mappings': [{'allele_string': 'C/G',\n",
      "                'ancestral_allele': 'C',\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133424434,\n",
      "                'location': '9:133424434-133424434',\n",
      "                'seq_region_name': '9',\n",
      "                'start': 133424434,\n",
      "                'strand': 1},\n",
      "               {'allele_string': 'C/G',\n",
      "                'ancestral_allele': None,\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133424582,\n",
      "                'location': 'CHR_HG2030_PATCH:133424582-133424582',\n",
      "                'seq_region_name': 'CHR_HG2030_PATCH',\n",
      "                'start': 133424582,\n",
      "                'strand': 1}],\n",
      "  'minor_allele': None,\n",
      "  'most_severe_consequence': 'missense_variant',\n",
      "  'name': 'rs121908467',\n",
      "  'source': 'Variants (including SNPs and indels) imported from dbSNP',\n",
      "  'synonyms': ['RCV000006154',\n",
      "               'VCV000005798',\n",
      "               '604134.0001',\n",
      "               'VAR_027111',\n",
      "               'NP_620595.1:p.His96Asp',\n",
      "               'NM_139027.5:c.286C>G',\n",
      "               'NP_620596.2:p.His96Asp',\n",
      "               'NM_139027.4:c.286C>G',\n",
      "               'NM_139026.4:c.286C>G',\n",
      "               'NP_620594.1:p.His96Asp',\n",
      "               'NM_139025.4:c.286C>G',\n",
      "               'NM_139026.5:c.286C>G'],\n",
      "  'var_class': 'SNP'},\n",
      " {'MAF': None,\n",
      "  'ambiguity': 'K',\n",
      "  'clinical_significance': ['pathogenic'],\n",
      "  'evidence': ['Frequency', 'Cited', 'Phenotype_or_Disease'],\n",
      "  'mappings': [{'allele_string': 'T/G',\n",
      "                'ancestral_allele': 'T',\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133448718,\n",
      "                'location': '9:133448718-133448718',\n",
      "                'seq_region_name': '9',\n",
      "                'start': 133448718,\n",
      "                'strand': 1},\n",
      "               {'allele_string': 'T/G',\n",
      "                'ancestral_allele': None,\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133448866,\n",
      "                'location': 'CHR_HG2030_PATCH:133448866-133448866',\n",
      "                'seq_region_name': 'CHR_HG2030_PATCH',\n",
      "                'start': 133448866,\n",
      "                'strand': 1}],\n",
      "  'minor_allele': None,\n",
      "  'most_severe_consequence': 'missense_variant',\n",
      "  'name': 'rs121908468',\n",
      "  'source': 'Variants (including SNPs and indels) imported from dbSNP',\n",
      "  'synonyms': ['VAR_027132',\n",
      "               'NM_139027.5:c.2851T>G',\n",
      "               'NM_139027.4:c.2851T>G',\n",
      "               'NP_620594.1:p.Cys951Gly',\n",
      "               'NM_139025.4:c.2851T>G',\n",
      "               'NM_139026.5:c.2758T>G',\n",
      "               'NP_620595.1:p.Cys920Gly',\n",
      "               'NP_620596.2:p.Cys951Gly',\n",
      "               'NM_139026.4:c.2758T>G',\n",
      "               '604134.0002',\n",
      "               'RCV000006155',\n",
      "               'VCV000005799'],\n",
      "  'var_class': 'SNP'},\n",
      " {'MAF': None,\n",
      "  'ambiguity': 'H',\n",
      "  'clinical_significance': ['pathogenic'],\n",
      "  'evidence': ['Frequency',\n",
      "               'Cited',\n",
      "               'Phenotype_or_Disease',\n",
      "               'TOPMed',\n",
      "               'gnomAD'],\n",
      "  'mappings': [{'allele_string': 'C/A/T',\n",
      "                'ancestral_allele': 'C',\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133424452,\n",
      "                'location': '9:133424452-133424452',\n",
      "                'seq_region_name': '9',\n",
      "                'start': 133424452,\n",
      "                'strand': 1},\n",
      "               {'allele_string': 'C/A/T',\n",
      "                'ancestral_allele': None,\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133424600,\n",
      "                'location': 'CHR_HG2030_PATCH:133424600-133424600',\n",
      "                'seq_region_name': 'CHR_HG2030_PATCH',\n",
      "                'start': 133424600,\n",
      "                'strand': 1}],\n",
      "  'minor_allele': None,\n",
      "  'most_severe_consequence': 'missense_variant',\n",
      "  'name': 'rs121908469',\n",
      "  'source': 'Variants (including SNPs and indels) imported from dbSNP',\n",
      "  'synonyms': ['VCV000005800',\n",
      "               'RCV000006156',\n",
      "               '604134.0003',\n",
      "               'NM_139026.4:c.304C>T',\n",
      "               'NP_620595.1:p.Arg102Cys',\n",
      "               'NM_139027.4:c.304C>A',\n",
      "               'NM_139027.4:c.304C>T',\n",
      "               'NP_620594.1:p.Arg102Ser',\n",
      "               'NP_620596.2:p.Arg102Ser',\n",
      "               'NM_139026.5:c.304C>A',\n",
      "               'NM_139027.5:c.304C>A',\n",
      "               'NM_139026.5:c.304C>T',\n",
      "               'NM_139025.4:c.304C>T',\n",
      "               'NP_620595.1:p.Arg102Ser',\n",
      "               'NP_620596.2:p.Arg102Cys',\n",
      "               'NM_139025.4:c.304C>A',\n",
      "               'NM_139026.4:c.304C>A',\n",
      "               'NM_139027.5:c.304C>T',\n",
      "               'NP_620594.1:p.Arg102Cys',\n",
      "               'VAR_027112'],\n",
      "  'var_class': 'SNP'},\n",
      " {'MAF': None,\n",
      "  'ambiguity': 'Y',\n",
      "  'clinical_significance': ['pathogenic'],\n",
      "  'evidence': ['Frequency',\n",
      "               'Cited',\n",
      "               'ESP',\n",
      "               'Phenotype_or_Disease',\n",
      "               'ExAC',\n",
      "               'TOPMed',\n",
      "               'gnomAD'],\n",
      "  'mappings': [{'allele_string': 'C/T',\n",
      "                'ancestral_allele': 'C',\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133426246,\n",
      "                'location': '9:133426246-133426246',\n",
      "                'seq_region_name': '9',\n",
      "                'start': 133426246,\n",
      "                'strand': 1},\n",
      "               {'allele_string': 'C/T',\n",
      "                'ancestral_allele': None,\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133426394,\n",
      "                'location': 'CHR_HG2030_PATCH:133426394-133426394',\n",
      "                'seq_region_name': 'CHR_HG2030_PATCH',\n",
      "                'start': 133426394,\n",
      "                'strand': 1}],\n",
      "  'minor_allele': None,\n",
      "  'most_severe_consequence': 'missense_variant',\n",
      "  'name': 'rs121908470',\n",
      "  'source': 'Variants (including SNPs and indels) imported from dbSNP',\n",
      "  'synonyms': ['604134.0004',\n",
      "               'RCV000006157',\n",
      "               'VCV000005801',\n",
      "               'NM_139026.4:c.587C>T',\n",
      "               'NM_139027.4:c.587C>T',\n",
      "               'NP_620596.2:p.Thr196Ile',\n",
      "               'NM_139026.5:c.587C>T',\n",
      "               'NM_139027.5:c.587C>T',\n",
      "               'NM_139025.4:c.587C>T',\n",
      "               'NP_620595.1:p.Thr196Ile',\n",
      "               'NP_620594.1:p.Thr196Ile',\n",
      "               'VAR_027114'],\n",
      "  'var_class': 'SNP'},\n",
      " {'MAF': None,\n",
      "  'ambiguity': 'R',\n",
      "  'clinical_significance': ['pathogenic'],\n",
      "  'evidence': ['Frequency', 'Cited', 'Phenotype_or_Disease', 'gnomAD'],\n",
      "  'mappings': [{'allele_string': 'G/A',\n",
      "                'ancestral_allele': 'G',\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133433478,\n",
      "                'location': '9:133433478-133433478',\n",
      "                'seq_region_name': '9',\n",
      "                'start': 133433478,\n",
      "                'strand': 1},\n",
      "               {'allele_string': 'G/A',\n",
      "                'ancestral_allele': None,\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133433626,\n",
      "                'location': 'CHR_HG2030_PATCH:133433626-133433626',\n",
      "                'seq_region_name': 'CHR_HG2030_PATCH',\n",
      "                'start': 133433626,\n",
      "                'strand': 1}],\n",
      "  'minor_allele': None,\n",
      "  'most_severe_consequence': 'missense_variant',\n",
      "  'name': 'rs121908471',\n",
      "  'source': 'Variants (including SNPs and indels) imported from dbSNP',\n",
      "  'synonyms': ['604134.0005',\n",
      "               'RCV000006158',\n",
      "               'VCV000005802',\n",
      "               'NP_620595.1:p.Arg367His',\n",
      "               'NM_139027.5:c.1193G>A',\n",
      "               'NM_139025.4:c.1193G>A',\n",
      "               'NP_620594.1:p.Arg398His',\n",
      "               'NM_139026.5:c.1100G>A',\n",
      "               'NP_620596.2:p.Arg398His',\n",
      "               'NM_139026.4:c.1100G>A',\n",
      "               'NM_139027.4:c.1193G>A',\n",
      "               'VAR_027119'],\n",
      "  'var_class': 'SNP'},\n",
      " {'MAF': 0.0002,\n",
      "  'ambiguity': 'K',\n",
      "  'clinical_significance': ['pathogenic'],\n",
      "  'evidence': ['Frequency',\n",
      "               '1000Genomes',\n",
      "               'Cited',\n",
      "               'ESP',\n",
      "               'Phenotype_or_Disease',\n",
      "               'ExAC',\n",
      "               'TOPMed',\n",
      "               'gnomAD'],\n",
      "  'mappings': [{'allele_string': 'T/G',\n",
      "                'ancestral_allele': 'T',\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133454440,\n",
      "                'location': '9:133454440-133454440',\n",
      "                'seq_region_name': '9',\n",
      "                'start': 133454440,\n",
      "                'strand': 1},\n",
      "               {'allele_string': 'T/G',\n",
      "                'ancestral_allele': None,\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133454588,\n",
      "                'location': 'CHR_HG2030_PATCH:133454588-133454588',\n",
      "                'seq_region_name': 'CHR_HG2030_PATCH',\n",
      "                'start': 133454588,\n",
      "                'strand': 1}],\n",
      "  'minor_allele': 'G',\n",
      "  'most_severe_consequence': 'missense_variant',\n",
      "  'name': 'rs121908472',\n",
      "  'source': 'Variants (including SNPs and indels) imported from dbSNP',\n",
      "  'synonyms': ['604134.0006',\n",
      "               'RCV000006159',\n",
      "               'VCV000005803',\n",
      "               'VAR_027133',\n",
      "               'NM_139027.4:c.3070T>G',\n",
      "               'NP_620596.2:p.Cys1024Gly',\n",
      "               'NM_139025.4:c.3070T>G',\n",
      "               'NM_139026.5:c.2977T>G',\n",
      "               'NP_620595.1:p.Cys993Gly',\n",
      "               'NP_620594.1:p.Cys1024Gly',\n",
      "               'NM_139027.5:c.3070T>G',\n",
      "               'NM_139026.4:c.2977T>G'],\n",
      "  'var_class': 'SNP'},\n",
      " {'MAF': None,\n",
      "  'ambiguity': 'R',\n",
      "  'clinical_significance': ['pathogenic'],\n",
      "  'evidence': ['Frequency', 'Cited', 'Phenotype_or_Disease', 'ExAC', 'gnomAD'],\n",
      "  'mappings': [{'allele_string': 'A/G',\n",
      "                'ancestral_allele': 'A',\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133437895,\n",
      "                'location': '9:133437895-133437895',\n",
      "                'seq_region_name': '9',\n",
      "                'start': 133437895,\n",
      "                'strand': 1},\n",
      "               {'allele_string': 'A/G',\n",
      "                'ancestral_allele': None,\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133438043,\n",
      "                'location': 'CHR_HG2030_PATCH:133438043-133438043',\n",
      "                'seq_region_name': 'CHR_HG2030_PATCH',\n",
      "                'start': 133438043,\n",
      "                'strand': 1}],\n",
      "  'minor_allele': None,\n",
      "  'most_severe_consequence': 'missense_variant',\n",
      "  'name': 'rs121908473',\n",
      "  'source': 'Variants (including SNPs and indels) imported from dbSNP',\n",
      "  'synonyms': ['NM_139027.4:c.1582A>G',\n",
      "               'NM_139025.4:c.1582A>G',\n",
      "               'NP_620594.1:p.Arg528Gly',\n",
      "               'NM_139026.4:c.1489A>G',\n",
      "               'NM_139027.5:c.1582A>G',\n",
      "               'NP_620596.2:p.Arg528Gly',\n",
      "               'NP_620595.1:p.Arg497Gly',\n",
      "               'NM_139026.5:c.1489A>G',\n",
      "               'VAR_027123',\n",
      "               'VCV000005804',\n",
      "               'RCV000006160',\n",
      "               '604134.0007'],\n",
      "  'var_class': 'SNP'},\n",
      " {'MAF': None,\n",
      "  'ambiguity': 'R',\n",
      "  'clinical_significance': ['pathogenic'],\n",
      "  'evidence': ['Frequency', 'Cited', 'Phenotype_or_Disease', 'gnomAD'],\n",
      "  'mappings': [{'allele_string': 'G/A',\n",
      "                'ancestral_allele': 'G',\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133456138,\n",
      "                'location': '9:133456138-133456138',\n",
      "                'seq_region_name': '9',\n",
      "                'start': 133456138,\n",
      "                'strand': 1},\n",
      "               {'allele_string': 'G/A',\n",
      "                'ancestral_allele': None,\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133456286,\n",
      "                'location': 'CHR_HG2030_PATCH:133456286-133456286',\n",
      "                'seq_region_name': 'CHR_HG2030_PATCH',\n",
      "                'start': 133456286,\n",
      "                'strand': 1}],\n",
      "  'minor_allele': None,\n",
      "  'most_severe_consequence': 'missense_variant',\n",
      "  'name': 'rs121908474',\n",
      "  'source': 'Variants (including SNPs and indels) imported from dbSNP',\n",
      "  'synonyms': ['604134.0010',\n",
      "               'VCV000005808',\n",
      "               'RCV000006164',\n",
      "               'VAR_027137',\n",
      "               'NM_139027.5:c.3470G>A',\n",
      "               'NM_139026.4:c.3377G>A',\n",
      "               'NM_139025.4:c.3638G>A',\n",
      "               'NM_139026.5:c.3377G>A',\n",
      "               'NM_139027.4:c.3470G>A',\n",
      "               'NP_620594.1:p.Cys1213Tyr',\n",
      "               'NP_620596.2:p.Cys1157Tyr',\n",
      "               'NP_620595.1:p.Cys1126Tyr'],\n",
      "  'var_class': 'SNP'},\n",
      " {'MAF': None,\n",
      "  'ambiguity': 'Y',\n",
      "  'clinical_significance': ['pathogenic'],\n",
      "  'evidence': ['Frequency',\n",
      "               'Cited',\n",
      "               'Phenotype_or_Disease',\n",
      "               'ExAC',\n",
      "               'TOPMed',\n",
      "               'gnomAD'],\n",
      "  'mappings': [{'allele_string': 'C/T',\n",
      "                'ancestral_allele': 'C',\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133442504,\n",
      "                'location': '9:133442504-133442504',\n",
      "                'seq_region_name': '9',\n",
      "                'start': 133442504,\n",
      "                'strand': 1},\n",
      "               {'allele_string': 'C/T',\n",
      "                'ancestral_allele': None,\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133442652,\n",
      "                'location': 'CHR_HG2030_PATCH:133442652-133442652',\n",
      "                'seq_region_name': 'CHR_HG2030_PATCH',\n",
      "                'start': 133442652,\n",
      "                'strand': 1}],\n",
      "  'minor_allele': None,\n",
      "  'most_severe_consequence': 'missense_variant',\n",
      "  'name': 'rs121908475',\n",
      "  'source': 'Variants (including SNPs and indels) imported from dbSNP',\n",
      "  'synonyms': ['VAR_027127',\n",
      "               'NM_139027.4:c.2074C>T',\n",
      "               'NP_620595.1:p.Arg661Cys',\n",
      "               'NM_139027.5:c.2074C>T',\n",
      "               'NM_139025.4:c.2074C>T',\n",
      "               'NP_620594.1:p.Arg692Cys',\n",
      "               'NP_620596.2:p.Arg692Cys',\n",
      "               'NM_139026.5:c.1981C>T',\n",
      "               'NM_139026.4:c.1981C>T',\n",
      "               '604134.0011',\n",
      "               'RCV000006165',\n",
      "               'VCV000005809'],\n",
      "  'var_class': 'SNP'},\n",
      " {'MAF': None,\n",
      "  'ambiguity': None,\n",
      "  'clinical_significance': ['pathogenic'],\n",
      "  'evidence': ['Cited', 'Phenotype_or_Disease'],\n",
      "  'mappings': [{'allele_string': 'T/TT',\n",
      "                'ancestral_allele': 'T',\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133456597,\n",
      "                'location': '9:133456597-133456597',\n",
      "                'seq_region_name': '9',\n",
      "                'start': 133456597,\n",
      "                'strand': 1},\n",
      "               {'allele_string': 'T/TT',\n",
      "                'ancestral_allele': None,\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133456745,\n",
      "                'location': 'CHR_HG2030_PATCH:133456745-133456745',\n",
      "                'seq_region_name': 'CHR_HG2030_PATCH',\n",
      "                'start': 133456745,\n",
      "                'strand': 1}],\n",
      "  'minor_allele': None,\n",
      "  'most_severe_consequence': 'frameshift_variant',\n",
      "  'name': 'rs387906341',\n",
      "  'source': 'Variants (including SNPs and indels) imported from dbSNP',\n",
      "  'synonyms': ['NP_620594.1:p.Leu1258fs',\n",
      "               'NM_139026.5:c.3509dup',\n",
      "               'NM_139027.4:c.3602dup',\n",
      "               'NP_620596.2:p.Leu1202fs',\n",
      "               'NM_139026.4:c.3509dup',\n",
      "               'NM_139027.5:c.3602dup',\n",
      "               'NP_620595.1:p.Leu1171fs',\n",
      "               'NM_139025.4:c.3770dup',\n",
      "               '604134.0008',\n",
      "               'VCV000005805',\n",
      "               'RCV000006161'],\n",
      "  'var_class': 'indel'},\n",
      " {'MAF': None,\n",
      "  'ambiguity': None,\n",
      "  'clinical_significance': ['pathogenic'],\n",
      "  'evidence': ['Cited', 'Phenotype_or_Disease'],\n",
      "  'mappings': [{'allele_string': 'CCAGCTGTGGCGCTGGAAACCTGCAACC/CC',\n",
      "                'ancestral_allele': 'CCAGCTGTGGCGCTGGAAACCTGCAACC',\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133443542,\n",
      "                'location': '9:133443515-133443542',\n",
      "                'seq_region_name': '9',\n",
      "                'start': 133443515,\n",
      "                'strand': 1},\n",
      "               {'allele_string': 'CCAGCTGTGGCGCTGGAAACCTGCAACC/CC',\n",
      "                'ancestral_allele': None,\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133443690,\n",
      "                'location': 'CHR_HG2030_PATCH:133443663-133443690',\n",
      "                'seq_region_name': 'CHR_HG2030_PATCH',\n",
      "                'start': 133443663,\n",
      "                'strand': 1}],\n",
      "  'minor_allele': None,\n",
      "  'most_severe_consequence': 'frameshift_variant',\n",
      "  'name': 'rs387906342',\n",
      "  'source': 'Variants (including SNPs and indels) imported from dbSNP',\n",
      "  'synonyms': ['NM_139027.5:c.2376_2401del',\n",
      "               'NP_620595.1:p.Ala762fs',\n",
      "               'NM_139026.5:c.2283_2308del',\n",
      "               'NM_139025.4:c.2376_2401del',\n",
      "               'NP_620596.2:p.Ala793fs',\n",
      "               'NM_139026.4:c.2283_2308del',\n",
      "               'NM_139027.4:c.2376_2401del',\n",
      "               'NP_620594.1:p.Ala793fs',\n",
      "               '604134.0009',\n",
      "               'RCV000006162',\n",
      "               'VCV000005806'],\n",
      "  'var_class': 'indel'},\n",
      " {'MAF': 0.005591,\n",
      "  'ambiguity': 'Y',\n",
      "  'clinical_significance': ['likely benign'],\n",
      "  'evidence': ['Frequency',\n",
      "               '1000Genomes',\n",
      "               'Cited',\n",
      "               'ESP',\n",
      "               'Phenotype_or_Disease',\n",
      "               'ExAC',\n",
      "               'TOPMed',\n",
      "               'gnomAD'],\n",
      "  'mappings': [{'allele_string': 'C/T',\n",
      "                'ancestral_allele': 'C',\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133442704,\n",
      "                'location': '9:133442704-133442704',\n",
      "                'seq_region_name': '9',\n",
      "                'start': 133442704,\n",
      "                'strand': 1},\n",
      "               {'allele_string': 'C/T',\n",
      "                'ancestral_allele': None,\n",
      "                'assembly_name': 'GRCh38',\n",
      "                'coord_system': 'chromosome',\n",
      "                'end': 133442852,\n",
      "                'location': 'CHR_HG2030_PATCH:133442852-133442852',\n",
      "                'seq_region_name': 'CHR_HG2030_PATCH',\n",
      "                'start': 133442852,\n",
      "                'strand': 1}],\n",
      "  'minor_allele': 'T',\n",
      "  'most_severe_consequence': 'missense_variant',\n",
      "  'name': 'rs41314453',\n",
      "  'source': 'Variants (including SNPs and indels) imported from dbSNP',\n",
      "  'synonyms': ['VCV000227176',\n",
      "               'RCV000220213',\n",
      "               'VAR_027128',\n",
      "               'NM_139027.4:c.2195C>T',\n",
      "               'NP_620595.1:p.Ala701Val',\n",
      "               'NM_139025.4:c.2195C>T',\n",
      "               'NM_139026.5:c.2102C>T',\n",
      "               'NM_139026.4:c.2102C>T',\n",
      "               'NP_620596.2:p.Ala732Val',\n",
      "               'NM_139027.5:c.2195C>T',\n",
      "               'NP_620594.1:p.Ala732Val'],\n",
      "  'var_class': 'SNP'}]\n"
     ]
    }
   ],
   "source": [
    "import pprint\n",
    "\n",
    "import requests\n",
    "\n",
    "server = \"https://rest.ensembl.org\"\n",
    "ext = \"/variation/human/pmid/11586351?\"\n",
    "headers = {\"Content-Type\": \"application/json\", \"Accept\": \"application/json\"}\n",
    "r = requests.get(server + ext, headers=headers)\n",
    "pprint.pprint(r.json())\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
